frontend/components/analytics/PixelPing.tsx

```diff
@@ -0,0 +1,24 @@
+import { PIXEL_PING_URL } from '@config/env';
+
+export const PixelPing = ({
+   type,
+   id,
+   site = 'ifixit',
+   lang = 'en',
+}: {
+   type: string;
+   id: string;
```

Comment:
I think `id` needs to be an `int` or the post will drop that hit.
https://github.com/iFixit/ifixit/blob/178253f6bb078761311a337d13a92bcd4d9dd584/Objects/PixelPingReader.php#L19

Reply:
Ah good catch. I'll update the types to help enforce this requirement.